### PR TITLE
adding formula for termtosvg

### PR DESCRIPTION
**Pull request by phaus**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by phaus** [comment]
Fixed… I guess the documentation was a little bit misleading there :-).

File: Formula/termtosvg.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  desc "Record terminal sessions as SVG animations"
+  homepage "https://nbedos.github.io/termtosvg"
+  url "https://github.com/nbedos/termtosvg/archive/0.6.0.tar.gz"
+  sha256 "09ccefb75a6a4f8be186b6efdc3cf355cf8de7dc2222b9e4d841d54e623dd261"
+
+  depends_on "python"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", "--upgrade", "setuptools"
```
Comment (fxcoudert) [inline Formula/termtosvg.rb]: Why `pip install --upgrade`?

Comment (phaus) [reply Formula/termtosvg.rb]: okay… I removed that one…

File: Formula/termtosvg.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    system libexec/"bin/pip", "install", "--upgrade", "pip"
+    system libexec/"bin/pip", "install", "-U", "-e", ".[dev]"
+    venv.pip_install_and_link buildpath
+    bin.install_symlink libexec/"bin/termtosvg"
+  end
+
+  test do
+    system bin/"termtosvg", "--help"
```
[the remark this answers is not in the document]
Comment (phaus) [reply Formula/termtosvg.rb]: I guess to ensure, that it is properly installed? Saw that in another Formula.
Is not test at all the better solution?

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/termtosvg.rb]: Yeah, this is to ensure a proper install. Having a test is a hard requirement to be added to homebrew nowadays though so you're going to need something.

[the remark this answers is not in the document]
Comment (phaus) [reply Formula/termtosvg.rb]: The thing is: I am not the developer of this Software. I just grabed an issues from their tracker. 
The goal of this formula is to install termtosvg, right?
So a test should prove, that the installation was successfull, right?

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/termtosvg.rb]: Yes, might be easiest to just record an `echo "Hello World"` using the tool and check if a file is created. It'd verify the functionality to the largest extend.

File: Formula/termtosvg.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/termtosvg.rb]: That's not what I meant. Why `pip install pip` and `setuptools` at all? They are provided by the python formula.

**Comment by phaus** [comment]
Removed `pip` and `setuptools` install.
Added a better test.

**Review by jgoldhammer — APPROVED** [review]

**Comment by phaus** [comment]
Do I have to do anything else @jgoldhammer  ?

**Comment by SMillerDev** [comment]
@fxcoudert got any comments still?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/termtosvg.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  desc "Record terminal sessions as SVG animations"
+  homepage "https://nbedos.github.io/termtosvg"
+  url "https://github.com/nbedos/termtosvg/archive/0.6.0.tar.gz"
+  sha256 "09ccefb75a6a4f8be186b6efdc3cf355cf8de7dc2222b9e4d841d54e623dd261"
+
+  depends_on "python"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", "-U", "-e", ".[dev]"
```
[the remark this answers is not in the document]
Comment (nbedos) [reply Formula/termtosvg.rb]: I'm not sure to understand what you mean by resources. If you're asking whether the Python package for termtosvg enforces a version number for each of its dependencies, the answer is no it does not.

`.[dev]` refers to a few dependencies [defined here](https://github.com/nbedos/termtosvg/blob/0.6.0/setup.py#L46) that are used during the development and testing phases. Those dependencies do not have to be installed on a user's machine so this part of the command can be omitted.

**Comment by phaus** [comment]
As @nbedos suggested, I updated the formula to 0.7.0. 
I can now use the build-in test suite.
I als removed the `[dev]` dependencies.

File: Formula/termtosvg.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    venv.pip_install_and_link buildpath
+    bin.install_symlink libexec/"bin/termtosvg"
+  end
+
+  test do
+    testfile = testpath/"test.svg"
+    exec("echo \"termtosvg-test\nexit\" | #{bin}/termtosvg  #{testfile} && grep -q termtosvg-test #{testfile}")
```
[the remark this answers is not in the document]
Comment (phaus) [reply Formula/termtosvg.rb]: I am now using the build-in test suite.

**Comment by phaus** [comment]
Merry chrismas. @fxcoudert  did you had a chance to have a look?

**Review by fxcoudert — APPROVED** [review]

**Comment by igas** [comment]
Thank you @phaus!

**Pull request closed without merging**